react-native/react/login/register/code-page/index.render.desktop.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+/*
+ * Screen to scan/show qrcode/text code. Goes into various modes with various options depending on if
+ * you're a phone/computer and if you're the existing device or the new device
+ */
+
+import React, {Component} from '../../../base-react'
+import {TextField, RaisedButton} from 'material-ui'
+import {codePageDeviceRoleExistingPhone, codePageDeviceRoleNewPhone,
+        codePageDeviceRoleExistingComputer, codePageDeviceRoleNewComputer} from '../../../constants/login'
+import {codePageModeShowCodeOrEnterText, codePageModeShowCode, codePageModeEnterText, codePageModeShowText} from '../../../constants/login'
+
+export default class CodePageRender extends Component {
+  renderText () {
+    return (
+      <div style={{display: 'flex', flexDirection: 'column', flex: 1, backgroundColor: 'green', alignItems: 'center', justifyContent: 'center', padding: 20}}>
+        <h1>Type this verification code into your other device</h1>
+        <p style={{backgroundColor: 'grey', padding: 20, marginTop: 20}}>{this.props.textCode}</p>
```

Comment:
What do you if textCode is null? if it can't be null it should be `isRequired` in the prop types